src/cli.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+import {execSync} from 'child_process';
+import * as path from 'path';
+import * as shim from './shim';
+
+// Parameters can be provided in the forms of:
+// $ rss googleapis
+// $ rss googleapis@20
+// $ rss googleapis@28.1.0
+// $ rss googleapis output.trace
+
+const args = process.argv.slice(2);
+if (args.length < 1) {
+  console.error(`USAGE: rss some-npm-module [output]`);
+  process.exit();
+}
+
+const mod = args[0];
+const [modName, modVer] = mod.split('@');
```

Comment:
Did you try something like `@google-cloud/common`? This mechanism fails that. Let's add this to the tests. Oh wait, this repo doesn't have tests yet 🤫.

I know you were trying to keep dependencies to a minimum, but let's just use this: https://www.npmjs.com/package/npm-package-arg.